lib/Chat/Parser/SystemMessage.php

```diff
@@ -505,10 +505,18 @@ protected function parseMessage(Message $chatMessage): void {
 				if (isset($metaData['caption']) && $metaData['caption'] !== '') {
 					$parsedMessage = $metaData['caption'];
 				}
-			} catch (\Exception $e) {
+			} catch (\Exception) {
+				$chatMessage->setMessageType(ChatManager::VERB_MESSAGE);
 				$parsedMessage = $this->l->t('{actor} shared a file which is no longer available');
 				if ($currentUserIsActor) {
 					$parsedMessage = $this->l->t('You shared a file which is no longer available');
+				} elseif ($currentActorType === Attendee::ACTOR_FEDERATED_USERS) {
+					$parsedMessage = $this->l->t('File shares are currently not supported in federated conversations');
+				}
+
+				$metaData = $parameters['metaData'] ?? [];
+				if (isset($metaData['caption']) && $metaData['caption'] !== '') {
+					$parsedMessage = '*' . $parsedMessage . "*\n\n" . $metaData['caption'];
```

Comment:
Maybe both variants should be italicized?
![image](https://github.com/nextcloud/spreed/assets/93392545/2e0462c1-4f5b-4968-90d2-2cbd6708f721)